openvasreporting/libs/parsed_data.py

```diff
@@ -179,7 +205,7 @@ def __init__(self, vuln_id, name, threat, **kwargs):
             raise TypeError("Expected basestring, got '{}' instead".format(type(level)))
         if not isinstance(tags, dict):
             raise TypeError("Expected dict, got '{}' instead".format(type(tags)))
-        if not isinstance(references, list):
+        if not isinstance(references, str):
             raise TypeError("Expected list, got '{}' instead".format(type(references)))
```

Comment:
Forgot to update error message after changing variable type:

    raise TypeError("Expected basestring, got '{}' instead".format(type(references)))

Reply:
agreed